server-mock/server/minipool-deposit-signature.go

```diff
@@ -31,16 +33,28 @@ func (s *NodeSetMockServer) minipoolDepositSignature(w http.ResponseWriter, r *h
 	data := apiv2.MinipoolDepositSignatureData{}
 
 	db := db.NewDatabase(s.logger)
-	privateKey, err := crypto.ToECDSA(db.ConstellationAdminPrivateKey)
+
+	adminAddress := crypto.PubkeyToAddress(db.ConstellationAdminPrivateKey.PublicKey).Hex()
```

Comment:
Same story here, don't need it to be a string since it isn't used as a string explicitly.

Reply:
updated